src/lib.rs

```diff
@@ -266,29 +270,79 @@ pub enum Locator<'a> {
 }
 
 impl<'a> From<Locator<'a>> for webdriver::command::LocatorParameters {
-    fn from(locator: Locator<'a>) -> webdriver::command::LocatorParameters {
+    fn from(locator: Locator<'a>) -> Self {
         match locator {
-            Locator::Css(s) => webdriver::command::LocatorParameters {
+            Locator::Css(s) => Self {
                 using: webdriver::common::LocatorStrategy::CSSSelector,
                 value: s.to_string(),
             },
-            Locator::Id(s) => webdriver::command::LocatorParameters {
+            Locator::Id(s) => Self {
                 using: webdriver::common::LocatorStrategy::XPath,
                 value: format!("//*[@id=\"{}\"]", s),
             },
-            Locator::XPath(s) => webdriver::command::LocatorParameters {
+            Locator::XPath(s) => Self {
                 using: webdriver::common::LocatorStrategy::XPath,
                 value: s.to_string(),
             },
-            Locator::LinkText(s) => webdriver::command::LocatorParameters {
+            Locator::LinkText(s) => Self {
                 using: webdriver::common::LocatorStrategy::LinkText,
                 value: s.to_string(),
             },
         }
     }
 }
 
-mod client;
+/// A [handle][1] to a browser window.
+///
+/// Should be obtained it via [`Client::window()`] method (or similar).
+///
+/// [1]: https://w3.org/TR/webdriver/#dfn-window-handles
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct WindowHandle(String);
```

Comment:
We should probably split these newtypes into their own file now that we're like to get more of them

Reply:
@jonhoo which module name would you prefer? I'm unsure here.

Reply:
`mod w3c` or `mod webdriver` maybe?